.github/workflows/images.yml

```diff
@@ -15,12 +23,9 @@ on:
     tags:
       - "v*"
 
-concurrency:
-  group: images-${{ github.ref }}
-  cancel-in-progress: true
-
```

Comment:
I had to remove this because while building for multiple versions of an implementation, whenever images.yml was being called from versioned-images.yml, the concurrent jobs were getting cancelled and only one of the version was getting built. Let me know if there's a way I could fix this. For now removing this did not cancel the concurrent jobs for building multiple versions.